app/uk/gov/hmrc/tradergoodsprofilesdatastore/controllers/DownloadDataSummaryController.scala

```diff
@@ -78,10 +81,26 @@ class DownloadDataSummaryController @Inject() (
         FileReadyUnseen,
         Some(FileInfo(notification.fileName, notification.fileSize, Instant.now, retentionDays, fileType))
       )
-      downloadDataSummaryRepository.set(summary).map { _ =>
-        //TODO send email
-        NoContent
-      }
+      downloadDataSummaryRepository
+        .set(summary)
+        .flatMap { _ =>
+          customsDataStoreConnector.getEmail(request.eori).flatMap {
+            case Some(email) =>
+              emailConnector
+                .sendDownloadRecordEmail(email.address, DownloadRecordEmailParameters(retentionDays))
+                .map { _ =>
+                  NoContent
+                }
+                .recover { case ex =>
+                  logger.error(s"Failed to send download record email: ${ex.getMessage}")
+                  BadRequest(s"Failed to send download record email")
```

Comment:
I would try to have the status code from the ex in the logging

Reply:
This is now removed